packages/claroty_ctd/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: 0.2.1
```

```suggestion
- version: 0.1.2
```
Can be a patch release because it's a bug fix rather than new functionality.